content/actions/learn-github-actions/introduction-to-github-actions.md

```diff
@@ -146,7 +146,7 @@ To help you understand how YAML syntax is used to create a workflow file, this s
   ```
 </td>
 <td>
-  Groups together all the steps that run in the <code>check-bats-version</code> job. Each line nested under this section is a separate action.
+  Groups together all the steps that run in the <code>check-bats-version</code> job. Each line nested under this section is a separate action or shell command.
```

Comment:
👋 Thanks a lot for this suggestion, @qmacro! Your change is great, but I'll make another teeny change to this sentence too. 

Although the workflow being described has only one line per step, generally each step isn't always only line (and mostly isn't). So, I've changed `line`->`item` to be a bit more correct too. 🙂 

```suggestion
  Groups together all the steps that run in the <code>check-bats-version</code> job. Each item nested under this section is a separate action or shell command.
```